mne/beamformer/_lcmv.py

```diff
@@ -748,12 +778,20 @@ def lcmv_epochs(epochs, forward, noise_cov, data_cov, reg=0.05, label=None,
     .. [2] Sekihara & Nagarajan. Adaptive spatial filters for electromagnetic
            brain imaging (2008) Springer Science & Business Media
     """
+    if picks is not None:
+        warn('Specifying picks is deprecated and will be removed in 0.16. '
+             'Specifying the selection of channels in info and setting picks '
+             'to None will remove this warning.',
+             DeprecationWarning)
+
+        epochs.info = pick_info(epochs.info, picks)
+
     # check whether data covariance is supplied
     _check_cov_matrix(data_cov)
 
     # construct spatial filter
     filters = make_lcmv(info=epochs.info, forward=forward, data_cov=data_cov,
-                        reg=reg, noise_cov=noise_cov, label=label, picks=picks,
+                        reg=reg, noise_cov=noise_cov, label=label, picks=None,
```

Comment:
since make_lcmv has never been released you should not have to have a picks params in it